dev-libs/zthread/zthread-2.3.2-r4.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit eutils autotools
```

Comment:
I was going to, but saw that -r3 had it that way, so I deferred to previous :). Fixed in next push.